presentation/components/src/presentation-components/tree/DataProvider.ts

```diff
@@ -197,14 +207,68 @@ export class PresentationTreeDataProvider implements IPresentationTreeDataProvid
   }
 }
 
+/** @alpha */
+export interface PresentationTreeNodeItem extends DelayLoadedTreeNodeItem {
+  key: NodeKey;
+  filterInfo?: PresentationInstanceFilterInfo;
+  isFilteringDisabled?: boolean;
```

Comment:
what do you think about grouping all filtering-related attributes together? e.g.
```ts
filtering?: {
  // filtering is disabled if descriptor or its getter is not set
  descriptor?: Descriptor | Promise<Descriptor>;
  // currently applied filter
  active?: PresentationInstanceFilterInfo;
};
```

Reply:
Updated. I like having getter or `Descriptor` itself on `TreeNodeItem`. Just changed `descriptor` property to not be nullable as there is no point of setting `filtering` object if `Descriptor` is `undefined`